src/main/kotlin/no/nav/tiltaksarrangor/repositories/AnsattRepository.kt

```diff
@@ -183,6 +183,28 @@ class AnsattRepository(
 		}
 	}
 
+	fun getAnsatt(personIdent: String): AnsattDbo? {
+		val ansattPersonaliaDbo = getAnsattPersonaliaDbo(personIdent) ?: return null
+		val ansattRolleListe = getAnsattRolleListe(ansattPersonaliaDbo.id)
+		val unikeRoller = ansattRolleListe.map { it.rolle.name }.distinct()
```

Comment:
Hvis vi finner de unike rollene her trenger man kanskje ikke å gjøre det i `KoordinatorService.getMineDeltakerlister`? Eller her?

Reply:
Grunnen til at jeg gjør det her er for å spare databasekall (ikke sjekke hvilke deltakerlister man har lagt til hvis man av en eller annen grunn ikke lenger har koordinatorrolle, og tilsvarende for veileder). Grunnen til at jeg også gjør det i koordinatorservicen er for at vi skal kunne sette koordinatorFor/veilederFor til null hvis man ikke har den ene eller andre rollen.

Reply:
Jeg hater at jeg må gjøre dette over flere spørringer :( Hvis det går an å unngå det gjør jeg det gjerne! Du får veldig mange rader tilbake selv med veldig få roller. Hvis jeg prøver denne 
```
SELECT * FROM ansatt
    INNER JOIN ansatt_rolle ar ON ansatt.id = ar.ansatt_id
LEFT JOIN koordinator_deltakerliste kd ON ansatt.id = kd.ansatt_id
LEFT JOIN veileder_deltaker vd ON ansatt.id = vd.ansatt_id
WHERE personident='fnr';
```
for Neil Armstrong (testbruker) for jeg 1200 rader. Blir ikke det mer enn vi klarer hvis vi får inn noen med mange tilganger i prod..?

Reply:
vi landet på at vi ser an litt om det blir et problem med flere spørringer. I så fall må vi vurdere å gjøre noe med datamodellen her også.